.github/workflows/release.yml

```diff
@@ -43,17 +43,25 @@ jobs:
           cd ${CURDIR}
           gh release upload --repo runtimeverification/k --clobber v${version} ${tarball}
 
-  nix-release:
-    name: 'Nix Release'
-    runs-on: ubuntu-20.04
-    environment: production
```

Comment:
This was only ever used for integration tests in the haskell-backend https://github.com/runtimeverification/haskell-backend/blob/858004428c6e9ff4a2e5733aee3b4d8909a3e13f/test.nix#L26-L36
but has long since been replaced with a different mechanism